magicctapipe/scripts/lst1_magic/IT_container_data_MC_bash_scripts/nsb_level_MC.py

```diff
@@ -0,0 +1,91 @@
+"""
+Bash scripts to run LSTnsb.py on all the LST runs by using parallel jobs
+Usage: python nsb_level.py (-c config.yaml)
+"""
+
+import argparse
+import glob
+import logging
+import os
+
+import numpy as np
+import yaml
+
+__all__ = ["bash_scripts"]
+
+logger = logging.getLogger(__name__)
+logger.addHandler(logging.StreamHandler())
+logger.setLevel(logging.INFO)
+
+
+def bash_scripts(run, config, source, env_name):
+    """Here we create the bash scripts (one per LST run)
+    Parameters
+    ----------
+    run: str
+        LST date and run number
+    config: str
+        Name of the configuration file
+    source: str
+        Target name
+    env_name:str
+        Name of the environment
+
+    """
+    lines = [
+        "#!/bin/sh\n\n",
+        "#SBATCH -p long\n",
+        "#SBATCH -J nsb\n",
+        "#SBATCH -N 1\n\n",
+        "ulimit -l unlimited\n",
+        "ulimit -s unlimited\n",
+        "ulimit -a\n\n",
+        f"time conda run -n  {env_name} LSTnsb_MC -c {config} -i {run} > {source}_nsblog_{run}.log 2>&1 \n\n",
```

Comment:
Just now I realized that run is actually of the form "YYYY_MM_DD,run"... it is a bit weird having them together, especially when passing them in the command line to `LSTnsb_MC` by being separated by a comma. It would be much more consistent and understandable if date and run are passed to this function and therefore to `LSTnsb_MC` separately.

Reply:
done